webapp/channels/src/components/file_preview_modal/file_preview_modal.tsx

```diff
@@ -31,8 +32,6 @@ import type {LinkInfo} from './types';
 
 import './file_preview_modal.scss';
 
-const PDFPreview = React.lazy(() => import('components/pdf_preview'));
```

Comment:
why did we remove lazy loading for this component?

Reply:
Lets add it back

Reply:
we didnt addressed this @Sudhanva-Nadiger

Reply:
it was giving a ts error ! i will look into it again

Reply:
@Sudhanva-Nadiger gentle ping on this one!